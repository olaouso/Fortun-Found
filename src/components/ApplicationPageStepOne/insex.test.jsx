import React from "react";
import renderer from "react-test-renderer";
import PostSectionStep1 from "./";

it("renders PostSectionStep1 correctly", () => {
  const tree = renderer.create(<PostSectionStep1 />).toJSON();
  expect(tree).toMatchSnapshot();
});
