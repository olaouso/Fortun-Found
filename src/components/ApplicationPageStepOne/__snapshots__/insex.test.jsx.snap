// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders PostSectionStep1 correctly 1`] = `
<div
  className="container"
>
  <div
    className="stepOneApplicationPage"
  >
    <div
      md={4}
    >
      <h2
        className="prerequisites"
      >
        applicationPageStep1.target
      </h2>
      <p
        className="prerequisitesDescription"
      />
      <input
        name="radiobutton"
        type="checkbox"
        value={true}
      />
      <label
        className=" prerequisitesDescription meet"
        htmlFor="radiobutton"
      >
        applicationPageStep1.MeetTarget
      </label>
    </div>
    <div />
  </div>
</div>
`;
